bindings/python/src/connection.pxi

```diff
@@ -291,7 +291,6 @@ cdef class Connection(object):
         def get_chunk():
             nonlocal rpc_result
 
```

Comment:
So this version doesn't work with the python bindings. I changed it to the following:
```suggestion
rpc_result = rpc_call_result(call)
```
And it seems to be working now. I don't know if the removes the 'fix' or not (except there is one less `rpc_retain` still...). I'll continue testing with the above fix.

Reply:
This is due to clumsiness on my part and doing development on the SOM and mac. The original line was:
rpc_result = rpc_retain(rpc_call_result(call))
It  looks like, after all my testing on one system, instead of just removing the call retain() I removed the whole line on the other. Your suggestion is in fact the intended code. Thanks!